### PR TITLE
add RELEASE-CHECKLIST.md

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <victorvieux@gmail.com>

### CODE REVIEW

**Comment by vieux** [comment]
ping @dongluochen

**Comment by abronan** [comment]
I'm glad to see my cheat-sheet improved and put into good use :D :dancer: 

Formatting went fubar though :ghost:

File: RELEASE-CHECKLIST.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Release Checklist
+
+### 1. Update version and CHANGELOG.md
+git checkout -b bump-<version>
+edit version/version.go
+edit CHANGELOG.md 
+git add .
+git push $GITHUBUSER bump-<version>
```
Comment (MHBauer) [inline RELEASE-CHECKLIST.md]: We're trying to make sure the push is to the user's github fork here, right?

File: RELEASE-CHECKLIST.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Release Checklist
+
+### 1. Update version and CHANGELOG.md
```
Comment (dongluochen) [inline RELEASE-CHECKLIST.md]: Add to the end 'on docker/swarm'. It's clear to us but not necessary to external maintainer.

File: RELEASE-CHECKLIST.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+git checkout -b bump-<version>
+edit version/version.go
+edit CHANGELOG.md 
+git add .
```
Comment (dongluochen) [inline RELEASE-CHECKLIST.md]: git commit -s

File: RELEASE-CHECKLIST.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+git rebase master
+git push origin
+git tag <tag>
+git push origin <tag>
+
+### 3. Update library image
+
+git clone git@github.com:docker/swarm-library-image.git
+cd swarm-library-image
+./update.sh <tag> (example: ./update.sh v0.2.0-rc2)
```
Comment (dongluochen) [inline RELEASE-CHECKLIST.md]: Validate build is successful.

File: RELEASE-CHECKLIST.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+fork https://github.com/docker-library/official-images.git
+
+git clone https://github.com/docker-library/official-images.git
+cd official-images
+git remote add $GITHUBUSER git@github.com:$GITHUBUSER/official-images.git
+git checkout -b update_swarm_<tag>
+edit library/swarm
+git add library/swarm
+git commit -s -m "update swarm <tag>"
+git push origin update_swarm_<tag>
```
Comment (dongluochen) [inline RELEASE-CHECKLIST.md]: I have 'git push $GITHUBUSER update_swarm_<tag>' from my log.

File: RELEASE-CHECKLIST.md (hunk 1 above)
Comment (vieux) [inline RELEASE-CHECKLIST.md]: yes, I'm open to suggestions if you know a better way

**Comment by vieux** [comment]
@abronan @dongluochen @MHBauer I addressed your comments, PTAL

**Comment by abronan** [comment]
LGTM

**Comment by dongluochen** [comment]
LGTM.

**Pull request merged**